### ISSUE TITLE
1.5.0-beta14. A problem of the starting of an appium node with numeric parameteres and old flags via command line.

### ISSUE THREAD

**Issue by TikhomirovSergey** [appium/appium#6141]
The sample of a command line and output:

```java
>node C:\The_Path_To_The_Appium_Node\node_modules\
appium\build\lib\main.js --device-ready-timeout 50

[Appium] Welcome to Appium v1.5.0-beta14
[Appium] Non-default server args:
[Appium]   deviceReadyTimeout: '50'
[Appium] Deprecated server args:
[Appium]   --device-ready-timeout => --default-capabilities '{"deviceReadyTimeou
t":"50"}'
[Appium] Default capabilities, which will be added to each request unless overri
dden by desired capabilities:
[Appium]   deviceReadyTimeout: '50'
[Appium] Appium REST http interface listener started on 0.0.0.0:4723
```

Ok. And then I'm trying to start this client code: 

```java
File appDir = new File("src/test/java/io/appium/java_client");
File app = new File(appDir, "ApiDemos-debug.apk");
DesiredCapabilities capabilities = new DesiredCapabilities();
capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "Android Emulator");
capabilities.setCapability(MobileCapabilityType.APP, app.getAbsolutePath());

AndroidDriver<MobileElement> driver = new AndroidDriver<>(new URL("http://0.0.0.0:4723/wd/hub"), 
capabilities);
```

The exception which is thrown: 

```java
org.openqa.selenium.SessionNotCreatedException: A new session could not be created. Details: The desiredCapabilities object was not valid for the following reason(s): deviceReadyTimeout must be of type number. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 790 milliseconds
Build info: version: '2.51.0', revision: '1af067dbcaedd7d2ab9af5151fc471d363d97193', time: '2016-02-05 11:20:57'
System info: host: '', ip: '', os.name: '', os.arch: 'amd64', os.version: '', java.version: '1.7.0_71'
Driver info: io.appium.java_client.android.AndroidDriver
```

Server output:
 https://gist.github.com/TikhomirovSergey/8fa8379643c9756eb3f7

Same things happen when I try to start node via command line (with some numeric parameters) and run the similar code on Mac OS X

**Comment by TikhomirovSergey** [comment]
appium/java-client#302
appium/appium-dotnet-driver#136

appium/java-client#319
appium/appium-dotnet-driver#138

**Comment by imurchie** [comment]
Fixed in 1.5 branch. We are waiting for a few things before publishing another version, but it should be available through source.

**Comment by TikhomirovSergey** [comment]
It is working now. 1.5.0-beta15
